mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -97,6 +99,140 @@ impl TendermintToken {
         };
         Ok(TendermintToken(Arc::new(token_impl)))
     }
+
+    pub fn ibc_withdraw(&self, req: IBCWithdrawRequest) -> WithdrawFut {
```

Comment:
Parts of this function and the `ibc_withdraw` function for `tendermint_coin` can probably be made into a common function, do you think this is a good idea?